### ISSUE TITLE
Dragging tab covers the bookmark folder I want to drop it on

### ISSUE THREAD

**Issue by ThePuterMan** [bwinton/TabCenter#673]
If I want to drag a tab to a bookmark folder, in this case one right on the left hand edge, when the bar pops out it covers the bookmark folder I want to drop it on. I tried to use Ctrl+Shift+L with one hand to close the bar whilst holding the other down on the mouse but it does not work when dragging.

Perhaps if a drag is initiated the bar could automatically shrink again. That would resolve the issue.

**Comment by ericawright** [comment]
if you click the icon in the top right of Tab Center,
<img width="44" alt="screen shot 2016-09-06 at 3 56 13 pm" src="https://cloud.githubusercontent.com/assets/10803178/19115938/cf8b08b6-8ae1-11e6-9d65-806b91f07deb.png">
it will remain "pinned" open. This will shift the content of your page rather than overlap it, you can then resize the drawer to the width you like. With it pinned like this, the bookmarks bar will no longer be overlapped.
Unfortunately, if you want it to remain unpinned I don't see an easy solution. I believe that most people who click and drag a tab are doing it to rearrange the order of their tabs, not drop them into the sidebar, and closing the drawer again would be inconvenient for those people.

**Comment by ThePuterMan** [comment]
I am not trying to drop a tab into the side bar, I want to drop one onto a BOOKMARK FOLDER. It happens that this folder is behind the tab bar when it springs open. Dragging to move the position of a tab is simply a matter of dragging within the bounds of the bounds of the tabs bar and in this case the bar should not close. It should only do so when a tab is dragged OUTSIDE the bounds of the tabs bar. This is a simple thing to achieve programmatically and I can't see why it should cause a problem.

I certainly don't want a great wide expanse at the side of the screen if I only have a few tabs open so either it has to be unpinned or I have to size it so that it just shows the icons which on a site with several pages the same is a problem as I can't see the headers.

**Comment by ericawright** [comment]
Yes, perhaps that is a solution, once you drag the mouse (and tab) out of bounds to minimize it.
(sorry, I sometimes call the bookmarks sidebar a "sidebar", as that is the element's id - but I believe I understand your intentions)

**Comment by ThePuterMan** [comment]
Well it would work for me, and I am sure I can't be the only one! And yes I think you  are understanding what I mean now. Here's hoping it will be implemented some time in the future.
